pymedphys/_gui/streamlit/mudensity/_config.py

```diff
@@ -0,0 +1,180 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
copyright?

Reply:
Yup, that was missed in copying it across. Fixed it up, in the PR stacked on top of this one.

...hopefully all these "PR" stackings was actually helpful and not super confusing...
keen for you feedback...